applications/main/nfc/scenes/nfc_scene_card_dump.c

```diff
@@ -0,0 +1,19 @@
+#include <furi.h>
+
+#include "../nfc_app_i.h"
+#include "../helpers/protocol_support/nfc_protocol_support.h"
+
+void nfc_scene_card_dump_on_enter(void* context) {
+    nfc_protocol_support_on_enter(NfcProtocolSupportSceneCardDump, context);
+}
+
+bool nfc_scene_card_dump_on_event(void* context, SceneManagerEvent event) {
+    UNUSED(context);
+    UNUSED(event);
+    return false;
+    //return nfc_protocol_support_on_event(NfcProtocolSupportSceneCardDump, context, event);
+}
```

Comment:
it's better to implement `nfc_protocol_support_on_event()` function and call it from here. Yes, the implementation is trivial, but it's better to place all logic in one place - nfc_protocol_support.c in our case

Reply:
Done